src/features/auto-resize-grid/js/auto-resize.js

```diff
@@ -41,6 +41,7 @@
             if (newGridHeight !== prevGridHeight || newGridWidth !== prevGridWidth) {
               uiGridCtrl.grid.gridHeight = newGridHeight;
               uiGridCtrl.grid.gridWidth = newGridWidth;
+              uiGridCtrl.grid.api.core.raise.gridHeightChanged(prevGridHeight, newGridHeight);
```

Comment:
this could be raised even if the gridWidth was the only thing that changed. I'd like to see a check here to only raise for height changes.

OR, you could change to gridDimensionChanged(prevGridHeight, newGridHeight, prevGridWidth, newGridWidth).  I like that even better.